cypress/support/util/ModalUtils.ts

```diff
@@ -6,8 +6,8 @@ export default class ModalUtils {
   private cancelModalBtn = "#modal-cancel";
   private closeModalBtn = ".pf-c-modal-box .pf-m-plain";
 
-  confirmModal() {
-    cy.get(this.confirmModalBtn).click();
+  confirmModal(force: boolean = false) {
```

Comment:
If you provide a default argument TypeScript can infer the type from the value. This means you can omit the type definition.

```suggestion
  confirmModal(force = false) {
```